sql/core/src/main/scala/org/apache/spark/sql/execution/command/AnalyzePartitionCommand.scala

```diff
@@ -110,7 +110,7 @@ case class AnalyzePartitionCommand(
       val newTotalSize = CommandUtils.calculateLocationSize(
         sessionState, tableMeta.identifier, p.storage.locationUri)
       val newRowCount = rowCounts.get(p.spec)
-      val newStats = CommandUtils.compareAndGetNewStats(tableMeta.stats, newTotalSize, newRowCount)
+      val newStats = CommandUtils.compareAndGetNewStats(p.stats, newTotalSize, newRowCount)
```

Comment:
is there different?

Reply:
Different. Stats in `tabeMeta` are table level stats, stats in `Partition` are partition level stats.